x/auth/src/genesis.rs

```diff
@@ -46,7 +46,7 @@ impl GenesisState {
             });
             Ok(())
         } else {
-            Err(AppError::Genesis(format!(
+            Err(GenesisError::Genesis(anyhow::anyhow!(
```

Comment:
Which sort of errors could happen durion genesis init?

Reply:
Addressed in a previous comment. Init genesis is different to `add_genesis_account`.